docs/_partials/0-20.migration.mdx

```diff
@@ -0,0 +1,539 @@
+
+If you want to continue using the open source image, you need to explicitly set the open source repository for your images.
+
+```yaml
+controlPlane:
+  statefulSet:
+    image:
+      repository: loft-sh/vcluster-oss
+```
+
+#### Supporting EKS distributions
+
+The v0.20 removed the support of EKS as a virtual cluster distribution.
+
+:::note reminder
+The Kubernetes distribution of the virtual cluster doesn't need to match the distribution of your host cluster. You can use any distribution of vCluster to deploy vCluster on EKS clusters.
+:::
+
+In order to continue using your virtual clusters that have EKS as the distribution, you can convert to using the k8s distribution and replace the images with the EKS images. While this was tested when v0.20.0 was released, it is not continuously tested. It's recommended spinning up new virtual clusters with the k8s distribution and migrate your workloads to them.
+
+```bash
+vcluster convert config --distro k8s -f /my/k8s/values.yaml > vcluster.yaml
+```
+
+After the conversion, you need to update the `vcluster.yaml` and change the images to be used. Refer to the EKS registries for [kube-apiserver](https://gallery.ecr.aws/eks-distro/kubernetes/kube-apiserver), [controller-manager](https://gallery.ecr.aws/eks-distro/kubernetes/kube-controller-manager) and [kube-scheduler](https://gallery.ecr.aws/eks-distro/kubernetes/kube-scheduler) for the specific tags you might want to use.
+
+**Example using EKS v1.30:**
+
+```yaml
+controlplane:
+  distro:
+    k8s:
+      enabled: true
+      apiserver:
+         image:
+           registry: "public.ecr.aws"
+           repository: "eks-distro/kubernetes/kube-apiserver"
+           tag: "v1.30.2-eks-1-30-10"
+      controllerManager:
+         image:
+           registry: "public.ecr.aws"
+            repository: "eks-distro/kubernetes/kube-controller-manager"
+            tag: "v1.30.2-eks-1-30-10"
+      scheduler:
+         image:
+           registry: "public.ecr.aws"
+           repository: "eks-distro/kubernetes/kube-scheduler"
+           tag: "v1.30.2-eks-1-30-10"
+```
+
+## Upgrade your vCluster instance
+
+:::warning Downgrading vCluster instances is NOT supported
+While downgrading to an older version of the vCluster CLI is supported, you cannot downgrade virtual cluster instances to a prior version of vCluster (for example, from 0.20+ back to 0.19). Make sure you have tested the upgrade sufficiently before rolling out this change to critical systems.
+:::
+
+
+<DeployChanges/>
+
+## Changes in the vCluster values between 0.19 and 0.20+
+
+The following is a reference for how specific 0.19 fields map to (or have changed in) 0.20+.
+
+autoDeletePersistentVolumeClaims
+<Columns>
+  <Column className='text--left'>
+  **0.19**:
+  ```yaml
+  autoDeletePersistentVolumeClaims: false
+  ```
+  </Column>
+  <Column className='text--left'>
+  **0.20+**:
+  ```yaml
```

Comment:
This code formatting is not displaying correctly in the github markdown preview, here or any below. Not sure how it shows up in docs.

Reply:
The mdx formatting doesn't show correctly on GH preview, you need to use the preview link in the PR description.